R/summary_skew_kurt.qmd

```diff
@@ -0,0 +1,176 @@
+for calculating Kurtosis.  Of the available packages, the functions in the **e1071** package provide the most 
+flexibility, and have options for three of the different methodologies.
+
+### e1071 Package {#e1071}
+
+The **e1071** package contains miscellaneous statistical functions from the Probability Theory Group at the Vienna University of Technology.
+The package includes functions for both Skewness and Kurtosis, and each function has a "type" parameter to specify the method.  There are 
+three available methods for Skewness, and three methods for Kurtosis. A portion of the documentation for these functions is included below:
+
+#### Skewness 
+
+The documentation for the `skewness()` function describes three types of skewness calculations:
+```{r, echo=FALSE, fig.align='center', out.width="80%"}
+knitr::include_graphics("../images/summarystats/skewness.png")
+```
+
+The three methods are illustrated in the following code:
+
+```{r eval=TRUE, echo = TRUE, message=FALSE, warning=FALSE}
+
+  library(e1071)
+
+  # Skewness - Type 1
+  e1071::skewness(dat$points, type = 1)
+  # [1] 0.9054442
+  
+    # Skewness - Type 2
+  e1071::skewness(dat$points, type = 2)
+  # [1] 1.009318
+  
+    # Skewness - Type 3
+  e1071::skewness(dat$points, type = 3)
+  # [1] 0.8164261
+```
+The default for the **e1071** `skewness()` function is Type 3, which will not match SAS.
+
+#### Kurtosis
+
+The documentation for the `kurtosis()` function describes three types of kurtosis calculations:
+```{r, echo=FALSE, fig.align='center', out.width="80%"}
+knitr::include_graphics("../images/summarystats/kurtosis.png")
+```
+
+The three methods are illustrated in the following code:
+
+```{r eval=TRUE, echo = TRUE, message=FALSE, warning=FALSE}
+
+  library(e1071)
+
+  # Kurtosis - Type 1
+  e1071::kurtosis(dat$points, type = 1)
+  # [1] -0.5833411
+  
+    # Kurtosis - Type 2
+  e1071::kurtosis(dat$points, type = 2)
+  # [1] -0.2991564
+  
+    # Kurtosis - Type 3
+  e1071::kurtosis(dat$points, type = 3)
+  # [1] -0.8948216
+```
+The default for the **e1071** `kurtosis()` function is Type 3, which will not match SAS.
+
+### Moments Package {#moments}
+
+The **moments** package is a well-known package with a variety of statistical functions.  The package contains functions for
+both Skewness and Kurtosis.  But these functions provide no "type" option.  The `skewness()` function in the **moments** package
+corresponds to Type 1 above.  The `kurtosis()` function uses a Pearson's measure of Kurtosis, which corresponds to none of the
+three types in the **e1071** package.
+
+
+```{r eval=TRUE, echo = TRUE, message=FALSE, warning=FALSE}
+
+  library(moments)
+
+  # Skewness - Type 1
+  moments::skewness(dat$points)
+  # [1] 0.9054442
+  
+  # Kurtosis - Pearson's measure
+  moments::kurtosis(dat$points)
+  # [1] 2.416659
+
+```
+Note that neither of the functions from the **moments** package match SAS.
```

Comment:
Documentation is not programming.  In documentation, redundancy is good.   I included this package because it is a popular package and want people to know that if they use it, it won't match SAS.